test/runnable_cxx/extra-files/test20652.cpp

```diff
@@ -0,0 +1,25 @@
+#if defined(__DMC__)
+
+// DMC doesn't support immintrin.h
+
+#else
```

Comment:
```suggestion
// DMC doesn't support immintrin.h
#if !defined(__DMC__)
```

For brevity ?

Reply:
On some extra thought over night, I'll just inline the appropriate typedef here, because imintrin won't be available everywhere, and notably non-x86.